src/game/Creature.cpp

```diff
@@ -132,7 +132,7 @@ Creature::Creature(CreatureSubtype subtype) : Unit(),
     m_lootStatus(CREATURE_LOOT_STATUS_NONE),
     m_corpseDecayTimer(0), m_respawnTime(0), m_respawnDelay(25), m_corpseDelay(60), m_aggroDelay(0), m_respawnradius(5.0f),
     m_subtype(subtype), m_defaultMovementType(IDLE_MOTION_TYPE), m_equipmentId(0),
-    m_AlreadyCallAssistance(false), m_AlreadySearchedAssistance(false),
+    m_AlreadyCallAssistance(false), m_AlreadySearchedAssistance(false), m_AllowCallAssistance(true),
```

Comment:
3 bool for callings assistances, i dont like that, its time to revamp it correctly. Add a structure, an enum or whatever to  unify call assistance parameters.